### PR TITLE
Set default event duration on the UI to 1 hour

### PR DESCRIPTION
**Pull request by hollyyuqizheng**

### CODE REVIEW

**Review by raulcruise — APPROVED** [review]

**Review by talvarez0507 — APPROVED** [review]

File: portfolio/src/main/webapp/calendar.js [hunk 1]
```diff
@@ -242,13 +243,16 @@ function setNewEventStartAndEndTimes() {
...
     var nextHour = now.getHours() + 1;
     defaultEventStartHour =
-        getClosestNextHour(nextHour, workHourStartString, workHourEndString);
+        getClosestNextHour(nextHour, workingHourStartString, workingHourEndString);
+    defaultEventEndHour = getClosestNextHour(nextHour+1, workingHourStartString, workingHourEndString);
   } else {
-    defaultEventStartHour = workHourStartString;
+    defaultEventStartHour = workingHourStartString;
+    const workingHourStart = parseInt(workingHourStartString.split(":")[0]);
+    defaultEventEndHour = getClosestNextHour(workingHourStart+1, workingHourStartString, workingHourEndString);
```
Comment (jamesnam-google) [inline portfolio/src/main/webapp/calendar.js]: nit: Space around +

File: portfolio/src/main/webapp/calendar.js [hunk 2]
```diff
@@ -242,13 +243,16 @@ function setNewEventStartAndEndTimes() {
     const now = new Date();
     var nextHour = now.getHours() + 1;
     defaultEventStartHour =
-        getClosestNextHour(nextHour, workHourStartString, workHourEndString);
+        getClosestNextHour(nextHour, workingHourStartString, workingHourEndString);
+    defaultEventEndHour = getClosestNextHour(nextHour+1, workingHourStartString, workingHourEndString);
```
Comment (jamesnam-google) [inline portfolio/src/main/webapp/calendar.js]: nit: Space around +

**Review by jamesnam-google — APPROVED** [review]

File: portfolio/src/main/webapp/calendar.js (hunk 1 above)
Comment (hollyyuqizheng) [reply portfolio/src/main/webapp/calendar.js]: Updated.

File: portfolio/src/main/webapp/calendar.js (hunk 2 above)
Comment (hollyyuqizheng) [reply portfolio/src/main/webapp/calendar.js]: Updated.

**Review by rgcase — APPROVED** [review]

**Pull request merged**